src/sentry/snuba/metrics/query_builder.py

```diff
@@ -189,9 +190,33 @@ def parse_query(query_string: str) -> Sequence[Condition]:
             },
         )
         where, _ = query_builder.resolve_conditions(query_string, use_aggregate_conditions=True)
+
```

Comment:
I could potentially edit https://github.com/getsentry/sentry/blob/c6b8a82722737c79ad37d670578dbc35e8276ffd/src/sentry/utils/snuba.py#L993-L1010 and the alias converter that changes to `project_id` to `project` but I felt this is a better approach since this will be hacked out eventually I guess
wdyt?

Reply:
Works for me